README.md

```diff
@@ -1,12 +1,14 @@
 # LaunchDarkly Sample Python Application 
 
- Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Python SDK reference guide](https://docs.launchdarkly.com/sdk/server-side/python).
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.
+
+Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Python SDK reference guide](https://docs.launchdarkly.com/sdk/server-side/python).
```

Comment:
```suggestion
We've built a console application that demonstrates how LaunchDarkly's SDK works.

Below, you'll find the basic build procedure. For more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Python SDK reference guide](https://docs.launchdarkly.com/sdk/server-side/python).
```
Felt like 'basic' and 'simple' in back-to-back paragraphs might be too much reassurance. :D

Reply:
@scribblingfox Are there other copyedit suggestions you have to the [example readme](https://launchdarkly.atlassian.net/wiki/spaces/ENG/pages/1499562073/SDK+hello+world#Example-README.md)? That way we can change there and the tickets in this epic can all use the same text.